### ISSUE TITLE
graficos

### ISSUE THREAD

**Issue by javier-develop** [uvejota/homeassistant-edata#5]
El grafico de consumo diario lo he personalizado un poco para leerlo mejor, aunque pierda el periodo de ajuste,
No le he puesto max ni min para que ajusten automáticamente los valores del eje Y.
Por si a alguien le sirve:

type: custom:apexcharts-card
graph_span: 30d
yaxis:
  - id: eje
    opposite: true
#  max: ~90
#  min: ~10
stacked: true
span:
  offset: '-1d'
header:
  show: true
  title: Consumo diario
  show_states: false
  colorize_states: false
all_series_config:
  type: column
  show:
    legend_value: false
series:
  - entity: sensor.edata_XXXX
    type: line
    yaxis_id: eje
    name: Total
    data_generator: |
      return hass.connection.sendMessagePromise({
      type: 'edata/consumptions/daily', 
      scups: 'XXXX'}).then(
          (resp) => {
              return resp.map((data, index) => {
                return [new Date(data['datetime']).getTime(), data['value_kWh']];
              });
          }
      );
    show:
      in_chart: true
      in_brush: false
  - entity: sensor.edata_XXXX
    name: Punta
    yaxis_id: eje
    data_generator: |
      return hass.connection.sendMessagePromise({
      type: 'edata/consumptions/daily', 
      scups: 'XXXX'}).then(
          (resp) => {
              return resp.map((data, index) => {
                return [new Date(data['datetime']).getTime(), data['value_p1_kWh']];
              });
          }
      );
  - entity: sensor.edata_XXXX
    name: Llano
    yaxis_id: eje
    data_generator: |
      return hass.connection.sendMessagePromise({
      type: 'edata/consumptions/daily', 
      scups: 'XXXX'}).then(
          (resp) => {
              return resp.map((data, index) => {
                return [new Date(data['datetime']).getTime(), data['value_p2_kWh']];
              });
          }
      );
  - entity: sensor.edata_XXXX
    name: Valle
    yaxis_id: eje
    data_generator: |
      return hass.connection.sendMessagePromise({
      type: 'edata/consumptions/daily', 
      scups: 'XXXX'}).then(
          (resp) => {
              return resp.map((data, index) => {
                return [new Date(data['datetime']).getTime(), data['value_p3_kWh']];
              });
          }
      );

**Comment by uvejota** [comment]
Hola! Gracias por tu aportación, me ha gustado este tipo de iniciativas, voy a mirar si puedo montar alguna wiki de github o algo así para colgar configuraciones :-)

En cualquier caso, me tomo la libertad de editarte el mensaje para "formatear" el yaml :-)

**Comment by uvejota** [comment]
Sí, hay algunos glitches gráficos, quizá este tipo de errores le interesan más al desarrollador de apexcharts-card, para solucionarlo yo puedo hacer poco :-)

**Comment by javier-develop** [comment]
ok gracias
